packages/react-components/src/templates/ComplianceReportForm.tsx

```diff
@@ -68,31 +76,58 @@ type ComplianceReportFormProps = {
   >;
 };
 
-type FormAction = 'cancel' | 'confirm' | '';
+type FormAction = 'cancel' | 'confirm' | 'confirmClosedStatus' | '';
+
+const manuscriptStatus = [
+  'Waiting for Report',
+  'Review Compliance Report',
+  'Manuscript Resubmitted',
+  'Submit Final Publication',
+  'Addendum Required',
+  'Compliant',
+  'Closed (other)',
+] as const;
+
+type ManuscriptStatus = (typeof manuscriptStatus)[number];
+
+const manuscriptStatusOptions: ReadonlyArray<Option<ManuscriptStatus>> =
+  manuscriptStatus.map((status) => ({
+    label: status,
+    value: status,
+  }));
 
 const getModalContent = (
-  formAction: FormAction,
+  formAction: FormAction | '',
 ): {
   title: string;
   content: string;
   confirmButtonText: string;
   confirmButtonStyle: 'primary' | 'warning';
 } =>
-  formAction && formAction === 'confirm'
-    ? {
-        title: 'Share compliance report?',
-        content:
-          'If you elect to share the compliance report, all associated team members (First Author(s), PM, PIs, Corresponding Author and Additional Authors) will receive a reminder on the CRN Hub and an email to notify them that this report is now available.',
-        confirmButtonText: 'Share Compliance Report',
-        confirmButtonStyle: 'primary',
-      }
-    : {
-        title: 'Cancel sharing of compliance report?',
-        content:
-          'Cancelling now will result in the loss of all entered data and will exit you from the sharing compliance report form.',
-        confirmButtonText: 'Cancel Compliance Report Sharing',
-        confirmButtonStyle: 'warning',
-      };
+  ({
+    confirm: {
+      title: 'Share compliance report?',
+      content:
+        'If you elect to share the compliance report, all associated team members (First Author(s), PM, PIs, Corresponding Author and Additional Authors) will receive a reminder on the CRN Hub and an email to notify them that this report is now available.',
+      confirmButtonText: 'Share Compliance Report',
+      confirmButtonStyle: 'primary' as const,
+    },
+    confirmClosedStatus: {
+      title:
+        'Share compliance report and set status to <compliant/closed (other)>?',
+      content:
+        'After you update the status to <compliant/closed (other)>, this change will be permanent and cannot be altered. If you need to make changes in the future, please reach out to the CMS admin. Additionally, by sharing the compliance report, all associated team members (First Author(s), PM, PIs, Corresponding Author and Additional Authors) will receive a reminder on the CRN Hub and an email to notify them that this report is now available.',
```

Comment:
<img width="978" alt="Screenshot 2025-03-27 at 10 25 18" src="https://github.com/user-attachments/assets/facbfb93-0f70-4394-991c-60dc89bdd5af" />

It's better to confirm with Padmini/João, but I believe this `<compliant/closed (other)>`should be replaced by `compliant` or `closed` depending on the value selected, like it is when we try to change the status in this page below

<img width="2294" alt="Screenshot 2025-03-27 at 10 26 38" src="https://github.com/user-attachments/assets/f302453b-044e-4957-a086-5fe010756bf6" />